Utils/data_analys.py

```diff
@@ -0,0 +1,52 @@
+import csv
+import pandas as pd
+from collections import Counter
```

Comment:
Let's firstly write imports starting with `import` keyword, then - starting with `from`... This version of code will look more readable :)